modules/stats.php

```diff
@@ -181,7 +181,8 @@ function stats_footer() {
 function stats_get_options() {
 	$options = get_option( 'stats_options' );
 
-	if ( !isset( $options['version'] ) || $options['version'] < STATS_VERSION )
+	if ( ( !isset( $options['version'] ) || $options['version'] < STATS_VERSION ) &&
```

Comment:
Since you're touching that old code, could you add in a space before `isset` to respect WP coding standards, as well as curly braces?